draft-ietf-ohai-ohttp.md

```diff
@@ -1250,7 +1257,7 @@ interception might choose to disable Oblivious HTTP in order to ensure that
 content is accessible to middleboxes.
 
 
-# Repurposing the Encapsulation Format
+# Repurposing the Encapsulation Format {#repurposing}
```

Comment:
I'm not a fan of the additional links to the repurposing section here, aside from this one :)